src/base/server_type.cpp

```diff
@@ -93,6 +124,17 @@ int init_server_type (const char *db_name)
   return er_code;
 }
 
+void setup_tran_server_params_on_single_node_config ()
+{
+  char *page_hosts_new_value;
+  constexpr size_t PAGE_HOSTS_BUFSIZE = 32;
+  page_hosts_new_value = (char *) malloc (PAGE_HOSTS_BUFSIZE);
```

Comment:
Explain why the newly allocated memory is not freed in a comment.
```suggestion
  page_hosts_new_value = (char *) malloc (PAGE_HOSTS_BUFSIZE); // free is called by sysprm_final()
```